bandwidth-file-spec.txt

```diff
@@ -591,7 +591,7 @@ num_target_relays=3862
 perc_measured_relays=46
 perc_measured_targed=60
 software=sbws
-software_version=0.8.1-dev0
+software_version=1.X.X
```

Comment:
[TODO: change the version of sbws that generates this when it is implemented]

Reply:
The Terminator is missing from the end of the header, and SHOULD be included.
Is this a bug in sbws?